python/src/expr.rs

```diff
@@ -91,6 +92,19 @@ impl Expr {
         Ok(Self { rust_expr, session })
     }
 
+    #[staticmethod]
+    #[pyo3(signature = (session, s, ns))]
+    fn literal_datetime(session: Session, s: i64, ns: i64) -> PyResult<Self> {
+        let mut rust_session = session.rust_session()?;
+
+        let rust_expr = rust_session
+            .add_literal(RustLiteral::Timedelta(s, ns))
```

Comment:
Should this be `TimeDelta`?

Reply:
Hm..I'm not sure. I usually think of it as one word, but it probably is two words. I like the way it looks as is, but feel free to disagree